velox/functions/prestosql/window/WindowFunctionsRegistration.cpp

```diff
@@ -31,6 +33,12 @@ void registerWindowFunctions() {
   window::registerPercentRank("percent_rank");
   window::registerCumeDist("cume_dist");
   window::registerNthValue("nth_value");
+
+  // Register all aggregate functions as window functions.
```

Comment:
When users register custom UDAFs, would they need to register these twice: once as aggregate function and another time as window function? Can we remove the requirement for registering agg function as window function?